setup.py

```diff
@@ -49,7 +49,7 @@ def check_env_flag(name: str, default=None) -> bool:
 PACKAGE_VERSION = os.environ.get("TORCH_MLIR_PYTHON_PACKAGE_VERSION") or "0.0.1"
 
 # If true, enable LTC build by default
-TORCH_MLIR_ENABLE_LTC_DEFAULT = True
+TORCH_MLIR_ENABLE_LTC_DEFAULT = False
```

Comment:
Why did you need to disable LTC? What build failures do you encounter using `clang`?